### DISCUSSION TITLE
Allow to open issues by commenting on code, like PR review

### DISCUSSION THREAD

**Discussion by minitauros** [community/community#9388] [Projects and Issues]
Sometimes I'm scrolling through some code and I would like to say something about it to a teammate. It would be too much work to open a PR to explain what I'm thinking. And I don't want to send them a direct message, because it's not pressing enough.

It would be amazing if I could browse the codebase in Github and add code comments like in a pull request review. I would then want to take all those code comments and open an issue with it and assign a team member.

**Comment by chriskuech** [comment]
This would be a game changer for me.  If you start working on a new project, you will want to let in "bad" code to get off the ground, but you will then want to audit the code later to identify all these issues.  This feature would be invaluable for that process and already sorely missed.

Additional improvements could be around highlighting "TODO" comments and suggesting to convert them to an issue.

**Reply by bradystroud** [reply]
> Additional improvements could be around highlighting "TODO" comments and suggesting to convert them to an issue.

Go give #58695 a ⬆️
